### PR TITLE
#163318638 Endpoint to delete a specific political party

### PR DESCRIPTION
**Pull request by perfect090**
**What does this PR do?**
Have an endpoint for deleting a specific political party

**Description of Task to be completed**
Have the following endpoint working
DELETE /api/v1/parties/:d/

**How should this be manually tested?**
They can be manually tested, using POSTMAN after cloning the repository

**What are the relevant pivotal tracker stories?**
#163318638

### CODE REVIEW

File: src/controllers/partyController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  static deletePartyById(req, res) {
+    const id = Number(req.params.id);
+    const deleteParty = parties.find(party => party.id === id);
+    const objId = parties.indexOf(deleteParty);
+    parties.splice(objId, 1);
+    if (objId) {
+      return res.status(200).json({
+        status: 200,
+        data: [{
+          id,
```
Comment (JCanaks) [inline src/controllers/partyController.js]: @perfect090 There is no need to send the id of the deleted party as a response

**Comment by perfect090** [comment]
Removed the id of the deleted party
screenshot 10

**Comment by JCanaks** [comment]
@perfect090 Work on improving your coverage

**Comment by perfect090** [comment]
> @perfect090 Work on improving your coverage

Resolved

**Pull request merged**